### DISCUSSION TITLE
git did not exit cleanly (exit code 128)

### DISCUSSION THREAD

**Discussion by x1774480** [community/community#46636] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Get error message when I try to execute Git push from local to master branch.

Thks for you help.

Regards.
Mathieu BONI
mathieu.boni-ext@socgen.com
![git error](https://user-images.githubusercontent.com/124694196/217267504-2bf565d6-7891-4720-a9e3-e09ad6b98aeb.png)

**Comment by SilasRodrigues19** [comment]
Hey @x1774480 

There is an answer on this forum with some instructions to fix it.

[Git push error "git did not exit cleanly (exit code 128)](https://qawithexperts.com/questions/139/git-push-error-git-did-not-exit-cleanly-exit-code-128-how-to)

**Comment by dbertels** [comment]
I did get this message when I made a 3 line comment when commiting to the local repository. I was able to push only after editing the file in .git/COMMIT_EDITMSG to just the one line.
